.github/file-filters.yml

```diff
@@ -92,7 +90,6 @@ migrations_modified:
 # These files will trigger our wokrflow to check if lockfile
 # updates are needed
 migration_lockfile:
-  - *backend_dependencies
```

Comment:
If django as a requirement changes, could there not be the possibility that it would impact the execution of migrations?

Reply:
backend_lintable now includes backend_dependencies.